target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -364,16 +364,27 @@ TARGET_DEVICES += tplink_cpe610-v2
 
 define Device/tplink-eap2x5
   $(Device/tplink-safeloader)
-  SOC := qca9563
   LOADER_TYPE := elf
   KERNEL := kernel-bin | append-dtb | lzma | loader-kernel
   KERNEL_INITRAMFS := $$(KERNEL)
   IMAGE/factory.bin := append-rootfs | tplink-safeloader factory | \
 	pad-extra 128
 endef
 
+define Device/tplink_eap225-wall-v2
+  $(Device/tplink-eap2x5)
+  SOC := qca9561
+  IMAGE_SIZE := 15104k
+  DEVICE_MODEL := EAP225-Wall
+  DEVICE_VARIANT := v2
+  TPLINK_BOARD_ID := EAP225-WALL-V2
```

Comment:
Consider putting TPLINK_BOARD_ID after DEVICE_PACKAGES to match the order of subsequent devices.

Reply:
Fixed.